src/Styleguide/Components/MarketInsights.tsx

```diff
@@ -1,46 +1,156 @@
 import { Sans } from "@artsy/palette"
+import { MarketInsightsArtistPage_artist } from "__generated__/MarketInsightsArtistPage_artist.graphql"
+import { highestCategory } from "Components/Artist/MarketInsights/MarketInsights"
 import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
 import { BorderBox, Box } from "Styleguide/Elements/Box"
 import { Flex } from "Styleguide/Elements/Flex"
 import { Responsive } from "Styleguide/Utils/Responsive"
 
 const wrapper = xs => props =>
   xs ? <Flex flexDirection="column" mb={1} {...props} /> : <Box {...props} />
 
-export interface MarketInsight {
-  primaryLabel: string
-  secondaryLabel: string
+export interface MarketInsightsProps {
+  artist: MarketInsightsArtistPage_artist
 }
 
-export interface MarketInsightsProps {
-  insights: MarketInsight[]
+const CATEGORIES = {
+  "blue-chip": "Blue chip",
+  "top-established": "Established",
+  "top-emerging": "Emerging",
+}
+const CATEGORY_LABEL_MAP = {
+  "blue-chip": "Represented by internationally recognized galleries.",
+  "top-established": "Represented by industry leading gallerie.s",
+  "top-emerging": "Represented by up-and-coming galleries.",
 }
 
 export class MarketInsights extends React.Component<MarketInsightsProps> {
+  renderAuctionHighlight(xs: boolean) {
+    const TextWrap = wrapper(xs)
+    if (
+      !this.props.artist.auctionResults ||
+      this.props.artist.auctionResults.edges.length < 1
+    ) {
+      return null
+    }
+    const topAuctionResult = this.props.artist.auctionResults.edges[0].node
```

Comment:
Do we need to be defensive at all about this data?

Reply:
No b/c of the early return above, we know that there's one `node` entry.

And then since we're sorting by price, we should have a record w/ a price, etc. So i think this should be good.